### DISCUSSION TITLE
What do you think of Discussions?

### DISCUSSION THREAD

**Discussion by chrisjsewell** [executablebooks/meta#89] [General]
GitHub Discussions is a new feature which is in [Beta development](https://github.blog/2020-05-06-new-from-satellite-2020-github-codespaces-github-discussions-securing-code-in-private-repositories-and-more/#discussions). We would like to use it as an alternative to issues, for more Q&A focussed or open-ended discussions (the clue is in the name!).
This would be an alternative to something like a [Google Groups](https://groups.google.com/forum/#!overview) mailing list or [Discourse Forum](https://discourse.jupyter.org/).
Let us know what your thoughts are and also you can provide feedback for GitHub discussions using the "Give feedback" button (it should appear on this page) or I am a member of the (private) `gh-community/discussions-feedback` repository, so can pass on feedback there 😄 
Thanks, Chris

**Comment by chrisjsewell** [comment]
cc @choldgraf @mmcky @AakashGfude @jstac @najuzilu

**Reply by chrisjsewell** [reply]
Note currently topic categories are hard-coded (that will probably change at some point). Also, there is an announcement feature which is visible in the demo but currently still being worked on and not available for trial

**Reply by jstac** [reply]
Looks nice, and good to keep things in one place, despite fear of Microsoft monopoly. thanks @chrisjsewell

**Reply by choldgraf** [reply]
I'm happy to give it a shot! I will hold off on creating something inside `discourse.jupyter.org` in that case and we can see what feels best for conversations here.

**Reply by mmcky** [reply]
Looks useful. It prefer it over another place to check (i.e. discourse)
